NextJob.md

```diff
@@ -0,0 +1,125 @@
+Next Job
+==================
+
+#### Table of Contents
+_______
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+- [Response structure](#response-structure)
+- [Versioning](#versioning)
+
+## Summary
+
+The Next Job provides skills related to a particular job and what are next level jobs available and the common skills between the present Job and the next level Jobs.It also tells us what are the missing skills a person should acquire to pursue the next level job. The Next Job is available at 
+`/core/careerpath/nextjob`.
```

Comment:
oh and like above please provide the full route to the service. And what fields are required and what not.